src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java

```diff
@@ -261,11 +261,12 @@ public InputStream getContent(String site, String path) throws ContentNotFoundEx
     public String writeContent(String site, String path, InputStream content) {
         // Write content to git and commit it
         String commitId = null;
-        String gitLockKey = SITE_SANDBOX_REPOSITORY_GIT_LOCK.replaceAll(PATTERN_SITE, site);
+        String gitLockKey = StringUtils.isEmpty(site) ?
+                                GLOBAL_REPOSITORY_GIT_LOCK :
+                                SITE_SANDBOX_REPOSITORY_GIT_LOCK.replaceAll(PATTERN_SITE, site);
```

Comment:
@phuongnq this would be a good time to use the methods from the helper to get the keys where possible

Reply:
`getRepoCacheKey`

Reply:
updated to use `getSandboxRepoLockKey` and `getPublishedRepoLockKey` methods